operator/validator/controller.go

```diff
@@ -1358,8 +1358,11 @@ func SetupCommitteeRunners(
 		}
 		config.ValueCheckF = valueCheckF
 
-		identifier := spectypes.NewMsgID(options.NetworkConfig.DomainType(), options.Operator.CommitteeID[:], role)
-		qbftCtrl := qbftcontroller.NewController(identifier[:], options.Operator, config, options.OperatorSigner, options.FullNode)
+		identifier := func() []byte {
+			identifier := spectypes.NewMsgID(options.NetworkConfig.AlanDomainType, options.Operator.CommitteeID[:], role)
+			return identifier[:]
+		}
```

Comment:
why do we need this to be a func?

Reply:
Also why `options.NetworkConfig.DomainType()` isn't good enough? since buildController is called everytime a new runner is created (on each slot with duty), we should be able to grab the correct domain when the fork epoch has passed.